### ISSUE TITLE
Hyperparameter for reproducing TSM RGB 8 frame

### ISSUE THREAD

**Issue by jd730** [epic-kitchens/action-models#11]
Hi,

Could you tell me a hyperparameter for reproducing the TSM result? I cannot find the hyperparameter in your workshop report and this repo.

Does it same as what have written in the TSM for other dataset?
i.e. imagenet pre-trained, lr 0.01, lr decay at 20, 40 epoch, training until 50 epoch, weight decay 1e-5?

Thank you.

**Comment by willprice** [comment]
Hi @jd730,
All models were trained in the same way.
![image](https://user-images.githubusercontent.com/966205/80809278-3dc8c580-8bb9-11ea-87ee-3622824bd91e.png)
Hope that helps :)
